processor/src/operations/sys_ops.rs

```diff
@@ -92,6 +92,19 @@ where
 
         Ok(())
     }
+
+    // CLOCK CYCLE
+    // --------------------------------------------------------------------------------------------
+
+    /// Pushes the current clock cycle onto the stack. The clock cycle is the number of times the
+    /// `execute` method has been called on the VM. The clock cycle is incremented after the
+    /// execution of each operation.
+    pub(super) fn op_clk(&mut self) -> Result<(), ExecutionError> {
+        let clk = self.system.clk();
+        self.stack.set(0, Felt::new(clk.into()));
```

Comment:
nit: can we just do `Felt::from(clk)` instead?

Reply:
thanks!